pkg/fileutil/lock_linux.go

```diff
@@ -50,7 +45,7 @@ var (
 func init() {
 	// use open file descriptor locks if the system supports it
 	getlk := syscall.Flock_t{Type: syscall.F_RDLCK}
```

Comment:
Why do you need to use dynamic probing for OFD locks ? This file is for Linux only.
Are OFD locks new in Linux?

Reply:
F_OFD_ locks are there since Linux kernel 3.15.

Reply:
Do you still support 3.15 and older?

Reply:
> Do you still support 3.15 and older?

@joakim-tjernlund I do not know about that, I am just a contributor here. Given the fact that this package is used by other software, I think it make sense to support older kernels.